### ISSUE TITLE
git bash windows fzf error

### ISSUE THREAD

**Issue by cloud-66** [ahmetb/kubectx#382]
![image](https://user-images.githubusercontent.com/41143306/222774088-50bd8ffc-4ada-42e7-963c-ecfbebc526ee.png)

Never mind how kubectx (golang version) installed, by downloading binary  or by scoop
kubectx (Enter)
it works in powershell, but don't  in git bash for windows. I guess because path should be in bash format  /c/Users/Cloud/scoop/shims/kubectx

**Comment by ahmetb** [comment]
Yeah this is an interesting case. We pass kubectx's full path to fzf, I wonder if we should if-else this for mingw.

**Comment by mloskot** [comment]
I'm observing the same problem while running `kubectx` in Git Bash session in Windows Terminal:

![image](https://user-images.githubusercontent.com/80741/230345279-99b5a8e3-7545-4d78-98a1-39bfe3261dde.png)

but if I switch to PowerShell in the same Windows Terminal, then it works:

![image](https://user-images.githubusercontent.com/80741/230345380-4c6fbb24-258b-4317-8266-8eb46765212b.png)

If I switch to plain Start > Git Bash, and not in Windows Terminal, then everything works:

![image](https://user-images.githubusercontent.com/80741/230345863-a8055edb-487a-4224-b6ec-079dc1d8604f.png)

```
$ kubectx --version
0.9.4
$ fzf --version
0.39.0 (2023040)
```

I tend to install everything with `winget`.

**Comment by mloskot** [comment]
Update to my https://github.com/ahmetb/kubectx/issues/382#issuecomment-1498810043

In Git Bash in Windows Terminal, interestingly, if I pipe `kubectx | fzf`:

![image](https://user-images.githubusercontent.com/80741/230591669-9601f525-f651-4e58-beee-166b1a5276ac.png)

then I get no error, but this

![image](https://user-images.githubusercontent.com/80741/230591818-55dbaf05-a30a-4ce1-9c07-a08e3e9dcdef.png)

But, selecting context and `ENTER` seems to have no effect.

**Comment by mloskot** [comment]
@cloud-66  See https://github.com/ahmetb/kubectx/issues/330#issuecomment-1506657091 for working workaround
